static/docs/command-reference/fetch.md

```diff
@@ -120,6 +120,8 @@ pipeline stages, as well as a few Git tags, such as our
 [get started example repo](https://github.com/iterative/example-get-started).
 Then we can see what happens with `dvc fetch` as we switch from tag to tag.
 
+#### [`▶ Try examples interactively`](https://katacoda.com/dvc/courses/examples/fetch)
```

Comment:
I still don't get why do make emphasis on fetch at all. It's one the most rare commands. Can we just delete/hide for now the interactive tutorial? remove it from the list of interactive tutorials? Otherwise when you google `dvc docs` now fetch is shown as one of the big sections

Reply:
> when you google `dvc docs` now fetch is shown as one of the big sections

Certainly there must be some other reasons, not the link to the interactive tutorial, since this change is not published yet.